### ISSUE TITLE
Add pitch accent data to dictionary

### ISSUE THREAD

**Issue by melink14** [melink14/rikaikun#169]
Useful for understanding pronunciation of words.

Not sure if there's a standard way to display pitch accents or a data source for them!

**Comment by melink14** [comment]
Some starters:
https://www.reddit.com/r/LearnJapanese/comments/2lhq9e/true_phonetic_dictionary_with_pitch_accent/
http://www.gavo.t.u-tokyo.ac.jp/ojad/
https://github.com/javdejong/nhk-pronunciation

Some epwing dictionaries probably have the data too if they're ever supported.

**Comment by melink14** [comment]
https://ankiweb.net/shared/info/278530045

**Comment by melink14** [comment]
A user sent me an email about Japanese dictionaries using a numeric indication for pitch which might be a more concise representation. Not sure if it's in free data sets or not.

> Recently I just would like to have a question that is it possible to add the NUMBER (see in the screenshot) which is the 'stressed position' of the pronunciation in Japanese, since it is very important for people to understand Japanese according to the tune.
> Inline image
> ![image](https://user-images.githubusercontent.com/1176550/90478171-b0a97c80-e167-11ea-865f-609b50defe5d.png)
> 
> 
> This Number, is usually very common in Japanese dictionary for every word.

**Comment by zkat** [comment]
I like how rikaichamp handles this: https://github.com/birtles/rikaichamp/issues/48#issuecomment-725774606

**Comment by melink14** [comment]
Yeah, the line based approach is pretty common and I had been following that issue as well.

Some notes going forward:
https://github.com/toasted-nutbread/yomichan-pitch-accent-dictionary is a processor of some raw data.

**Comment by melink14** [comment]
https://github.com/mifunetoshiro/kanjium
